inference_rules.adoc

```diff
@@ -261,14 +261,36 @@ Each Edge benchmark *requires* the following scenarios, and sometimes permit an
 
 |===
 |Area |Task |Required Scenarios |Optional Scenarios
-|Vision |Image classification |Single Stream, Offline |Multi-stream
-|Vision |Object detection (large) |Single Stream, Offline |Multi-stream
-|Vision |Object detection (small) |Single Stream, Offline |Multi-stream
+|Vision |Image classification |Single Stream, Offline |Multi-Stream
+|Vision |Object detection (large) |Single Stream, Offline |Multi-Stream
+|Vision |Object detection (small) |Single Stream, Offline |Multi-Stream
 |Vision |Medical image segmentation |Single Stream, Offline | N/A
 |Speech |Speech-to-text |Single Stream, Offline | N/A
 |Language |Language processing |Single Stream, Offline | N/A
 |===
 
+For Edge submitters' convenience, they are allowed to obtain only a Single Stream result
+and have Offline and optionally Multi-Stream results inferred according to the following rules:
+
+- The corresponding Offline result can be _inferred_ as the Queries per Second reported
+in the Single Stream result, that is 1000 divided by the Mean latency.  For example, a Single
```

Comment:
I think this sentence is easier to read if, "can be _inferred_ as the Queries per Second reported in the Single Stream result, that" is removed.

Reply:
Do you mean like this: " The corresponding Offline result can be _inferred_ as 1000 divided by the Mean latency in milliseconds"?

Reply:
"The corresponding Offline result is 1000 divided by the Mean latency in milliseconds."

Reply:
Done!